### ISSUE TITLE
Adding an ID column to permissions table

### ISSUE THREAD

**Issue by scottybo** [JosephSilber/bouncer#374]
This is an issue to do with compatibility with the newly released Laravel Nova.

It seems that if we're using a Postgres database alongside Nova, attempting to use a BelongsToMany relationship with the intermediate `permissions` table returns an error that no id column exists. See a similar issue mentioned here: https://github.com/laravel/nova-issues/issues/433

Is there a simple way to add the ID column to the Permissions table *without* altering the database? It seems there is no Model for Permission in bouncer, and trying to create one with the following structure doesn't seem to work (in terms of "faking" the id column).

Basically, I'm trying to tell Laravel to use the `ability_id` column as the `id` column.

```php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Permission extends Model {
    
    public $appends = [
        'id',
    ];
    
    public function getIdAttribute()
    {
        return $this->attributes['ability_id'];
    }
    
}
```

**Comment by JosephSilber** [comment]
Hmm. Not sure. Is there any reason we can't update our migration file to add an ID column?
